CHANGELOG.md

```diff
@@ -6,10 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased-`x.y.z`] - 2020-xx-xx
 
+### New Known Issues:
+- Some files that were included in Unreal Engine `4.23.x` [are not in](https://forums.unrealengine.com/unreal-engine/announcements-and-releases/1695917-unreal-engine-4-24-released?p=1715142#post1715142) `4.24.3`. This causes errors when you upgrade from `4.23.x` to `4.24.3`. You can resolve this issue by running `git restore .`.
```

Comment:
I would reword this slightly, to not be misleading.
```suggestion
- After upgrading to 4.24.3 using git, you may be left in a state where several .isph and .ispc files are missing, producing [compile errors](https://forums.unrealengine.com/unreal-engine/announcements-and-releases/1695917-unreal-engine-4-24-released?p=1715142#post1715142). You can fix this by running `git restore .`.
```

Reply:
Because I think these files were introduced with 4.24.0, Epic somehow deleted them in 4.24.1 and reintroduced them in 4.24.2, but the upgrade process can miss it for some obscure reason.

Reply:
much clearer, thanks